CHANGELOG.md

```diff
@@ -1,6 +1,27 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## Unreleased
+
+#### Added
+- `PanicPrinter::format_trace_to_string`
+
+#### Changed
+- Rename `Settings` -> `PanicPrinter`
```

Comment:
I think BacktracePrinter might be a better name here, because it seems to be handling printing panics and non-panic backtraces, when I first went to use this change I got very confused because I thought it only had formatting support form within the panic handler.

Reply:
Good point, I'll change this!